bots/lemmy_mlb_game_threads/__init__.py

```diff
@@ -6339,3 +6324,13 @@ def bot_state(self):
 
         self.log.debug("Bot Status: {}".format(botStatus))  # debug
         self.bot.detailedState = botStatus
+
+    def _truncate_post(self, text):
+        warning_text = '\n# Truncated: 10k Limit Reached'
+        max_length = self.settings["Lemmy"]["POST_CHARACTER_LIMIT"] - len(warning_text)
+        if len(text) >= max_length:
+            new_text = text[0:max_length - 1]
```

Comment:
We need to truncate to `max_length - len(warning_text) - 1`. Otherwise the user would have to do that math ahead of time and set the max_length appropriately.

Reply:
This is why max_length subtracts the length of warning_text, so you can edit the warning string and not worry about the length.